Numpy.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+# 1
+arr_0_to_20 = np.array(range(21))
+arr_0_to_20_after_scale = np.concatenate((arr_0_to_20[:8], (arr_0_to_20[9:16] * -1), arr_0_to_20[16:]))
```

Comment:
There is no need to concatenate and create a new numpy array. Using broadcasting and numpy indexing this can all be done quite elegantly.
https://numpy.org/doc/stable/user/basics.broadcasting.html
https://www.geeksforgeeks.org/numpy-indexing/